package.json

```diff
@@ -134,19 +134,18 @@
     },
     "win": {
       "icon": "./resources/build/icon.ico",
+      "artifactName": "${productName}-${os}-${env.NODE_ARCHITECTURE}-${version}-portable.${ext}",
```

Comment:
# Not possible

as this package.json creates ..
1. `.exe` (portable)
2. `.exe` (installer)
3. `.7z` (portable)  [ i changed it from `.zip` to `.7z` as this gives more compression ratio 80MB(`.zip`) to 50MB(`.7z`) ]

electron builder dosent provide the the ArtifactName attribute for `.7z` or `.zip` they  will use the default ArtifactName only provided under __win__ attribute.

## but for a cleaner look i have changed 

__now it will generate...__
- `Buttercup-win-{{arch}}-{{version}}-installer.exe`
- `Buttercup-win-{{arch}}-{{version}}-portable.exe`
- `Buttercup-win-{{arch}}-{{version}}.7z`

cant leave the `.7z` file as it give 2x to 3x speed at startup than the normal portable `.exe` :smile: 

i have also released them you can test them from [here](https://github.com/web-cpu-test/buttercup-desktop/releases/tag/patch-1) although i have tested the portable `.7z` .

names of the mac releases are not correct as i have not released them through electrobn builder so `fix-mac-zip.js` dosent run. but this is no problem or :bug:

Reply:
Thanks, that makes sense. I'm happy with that approach.

Mac builds run fine on my side, so I'm sure it'll be fine.